packages/web-components/src/components/ic-data-row/ic-data-row.css

```diff
@@ -55,3 +55,11 @@
   width: 128px;
   margin-bottom: var(--ic-space-xs);
 }
+
+:host(.breakpoint-xs) .value {
+  max-width: 75vw;
```

Comment:
I think this should be `75%` of the parent container not 75% off the viewport.

Reply:
I've updated it to 95% as 75 seemed too squished when converting from vw